modules/nw-ovn-kubernetes-session-affinity.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+:_content-type: CONCEPT
+[id="nw-ovn-kubernetes-session-affinity_{context}"]
+= Session affinity
+Session affinity is a feature that applies to kubernetes objects of type `services`. You can use `session affinity` if you want to ensure that each time you connect to a ServiceVIP:Port the traffic is always load balanced to the same backend. For more information, including how to set session affinity based on a client's IP address, refer to link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-affinity[Session affinity].
+
+[discrete]
+[id="nw-ovn-kubernetes-session-affinity-stickyness-timeout_{context}"]
+== Stickyness timeout for session affinity
+The OVN-Kubernetes network plugin for {product-title} calculates the stickyness timeout for a `session` from a client based on the last packet. For example, if you run a curl 10 times, the stickyness timer starts from 10th packet not 1st. As a result if the client is continuously contacting the service, then the session will never timeout. The timeout kicks in when the service has not received a packet for link:https://kubernetes.io/docs/reference/networking/virtual-ips/#session-stickiness-timeout[`timeoutSeconds`].
```

Comment:
s/stickyness/stickiness ?
s/from 10th packet not 1st/from 10th packet not first (Not sure what the rules are, and I looked. But 1st looks weird)

Reply:
The first one I'm not sure about https://redhat-documentation.github.io/supplementary-style-guide/#sticky-session. Second is a solid miss on my part, thank you.